### DISCUSSION TITLE
SHA-3 Keccak Vulnerability

### DISCUSSION THREAD

**Discussion by qnfm** [XKCP/XKCP#118] [General]
Dear all,
From this [paper](https://eprint.iacr.org/2023/331), it point out a vulnerability in the code package. Any plan to do about it ? (if vulnerability do exist)

**Comment by gvanas** [comment]
We fixed this in October last year in commit [fdc6fef0](https://github.com/XKCP/XKCP/commit/fdc6fef075f4e81d6b1bc38364248975e08e340a). See also the [security advisory](https://github.com/XKCP/XKCP/security/advisories/GHSA-6w4m-2xhg-2658) for more details.

**Reply by qnfm** [reply]
Nice, thanks for reply
